llm/requirements.txt

```
@@ -4,6 +4,8 @@ torch==1.13.0
 mosaicml==0.12.0
 mosaicml-streaming==0.1.2
 flash-attn==0.2.2
+# need a newer version of triton
+triton==2.0.0.dev20221103
```

```suggestion
triton==2.0.0.dev20221202
```

Any reason not to use the latest version?

---

I copied it from the [bert requirements](https://github.com/mosaicml/examples/blob/main/bert/requirements.txt#L8)